CollAction/Services/MessageServices.cs

```diff
@@ -31,7 +31,7 @@ public AuthMessageSender(IOptions<AuthMessageSenderOptions> authOptions, ILogger
 
             SendEmailRequest emailRequest = new SendEmailRequest()
             {
-                Source = _authOptions.FromAddress,
+                Source = $"Team CollAction <{_authOptions.FromAddress}>",
```

Comment:
Oh, this is not a good idea. 

We already have a configuration option "FromAddress". We just have to configure it as "Team CollAction <teamcollaction@gmail.com>".

This way:
* We can change it more easily afterwards by just changing a config parameter
* The code is the same for freonen and collaction (no divergence)